### PR TITLE
removing untruths from CQ doc

### PR DESCRIPTION
**Pull request by beckettsean**
What's left passes the parser so I assume it's valid, although I don't have a great dataset to test with right now.

### CODE REVIEW

**Comment by toddboom** [comment]
looks good to me aside from the one issue i flagged - if it parses, i'm guessing it doesn't actually do anything. that's one of the dream features. ;)

File: source/docs/v0.9/concepts/continuous_queries.md [hunk 1]
```diff
@@ -2,64 +2,30 @@
...
-
-When points are fanned out, the resulting points have the same `time` and `sequence_number`. That means if you know the source series, you'll be able to query the point that produced a fanout point by time and sequence number.
-
 ### Downsampling Continuous Queries
 
 This is expected to be the primary use case for continuous queries. When a continuous query is created from a select query that contains a `GROUP BY` time() clause, InfluxDB will write the aggregate into the target time series when each time interval elapses. On creation, the cluster will backfill old data asynchronously in the background.
 
 ```sql
-SELECT COUNT(name) FROM clicks GROUP BY time(1h) INTO clicks.count.1h
+SELECT COUNT(name) INTO clicksCount_1h FROM clicks GROUP BY time(1h)
```
Comment (beckettsean) [inline source/docs/v0.9/concepts/continuous_queries.md]: It only works that way, the INTO clause must directly follow the SELECT clause or the parser throws an error.

Comment (beckettsean) [inline source/docs/v0.9/concepts/continuous_queries.md]: Actually, I think you mean does that query work at all, and I believe the answer is no. Let me check. I might have just fixed the syntax without checking the function.

Comment (beckettsean) [inline source/docs/v0.9/concepts/continuous_queries.md]: Actually no that doesn't work at all, good catch.

**Pull request merged**